include/grpc++/server.h

```diff
@@ -138,10 +139,17 @@ class Server final : public ServerInterface, private GrpcLibraryCodegen {
   ///
   /// \param sync_cq_timeout_msec The timeout to use when calling AsyncNext() on
   /// server completion queues passed via sync_server_cqs param.
+  ///
+  /// \param thread_creator The thread creation function for the sync
+  /// server. Typically gpr_thd_new
```

Comment:
add a \param comment for `thread_joiner` too

Reply:
Done! Thanks.